app/main.py

```diff
@@ -2,0 +1,25 @@
+import os
+import shutil
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
+    if len(parts) != 3 or parts[0] != "mv":
+        raise ValueError(
+            "Команда повинна бути у форматі 'mv source destination'"
+        )
+    source = parts[1]
+    destination = parts[2]
+
+    if not os.path.exists(source):
+        raise FileNotFoundError(f"Файл {source} не існує")
+
+    if destination.endswith("/"):
+        os.makedirs(destination, exist_ok=True)
```

Comment:
Code Efficiency: You are creating a directory if it doesn't exist, which is correct. But you should also handle the case when the directory already exists. The 'os.makedirs' function will raise an 'FileExistsError' if the directory already exists and 'exist_ok' is False.